officeInsightIndex.js

```diff
@@ -7,6 +7,9 @@ var MEETING_ROOM_AVAILABILITY = '/rest/meetingRoom/availability/FIFTY_ONE';
 var MEETING_ROOM_AVAILABILITY_TIME = '?startTime=';
 var MEETING_ROOM_ASSISTANCE = '/rest/meetingRoom/assistance';
 var MEETING_ROOM_BOOK = '/rest/meetingRoom/book/';
+var GTFO_HOST = 'acheron.herokuapp.com';
```

Comment:
Hmm, ok, this is indicative of us needing some build process I guess. I can work on that separately some other time.